data/seeds/003-assignments.js

```diff
@@ -1,19 +1,15 @@
 const dummyData = [
   {
-    mentor_id: '7',
-    mentee_id: '10',
+    mentor_id: 'e1465095-e695-4e02-8fb2-19ed5bf5a132',
```

Comment:
was assignments a necessary cleanup for this ticket? Not too worried about the work done here just curious is ya'll got scope crept.

Reply:
I think you're right; we didn't need to make any changes here. I believe we were changing this file in an attempt to prop up the seeds/migrations. Our changes to the profiles table breaks most of the seed files because of the foreign key constraints. When this PR is merged, the migrations will work but not the seed files. This PR depends on some follow-up to clean out the rest of the unnecessary tables in Postgres.